font.css

```diff
@@ -0,0 +1,46 @@
+@font-face {
+  font-family: 'montserratbold';
+  src: 
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)